docs/user-installation-checklist.md

```diff
@@ -0,0 +1,77 @@
+# OpenShift Provider Certification Tool - Cluster Installation Checklist
+
+<!--
+Do not change the following markdown commented lines.
+__version__: 0.1.0-preview
+-->
+
+This checklist should be provided for every new support case, or when any items have been changed (for Example, Compute flavor). This will help the Red Hat engineers while working in your partner support case during the certification process.
+
+If you have any questions you can:
+
+- Review the [Installation Review Guide](./user-installation-review.md)
+- Review the [OpenShift Infrastructure Provider Guide](https://docs.providers.openshift.org/)
+- Review the [OpenShift Documentation Page related to the version your are certifying](https://docs.openshift.com/container-platform)
+- Ask your Red Hat Partner
+
+## Provider Information
+
+- Provider Name:                # Example: MyCloud
+- OpenShift Version:            # Example: 4.11.4
+- Certification Type:           # Options: (Standard | Upgrade)
```

Comment:
I would think each certification includes standard and upgrade. For example, certifying on the latest 4.11.z and also certifying upgrading from 4.11.z-1 -> 4.11.z

Reply:
Not sure if I follow your example. But I agree that this information may be duplicated as we already have this on the artifacts. The idea here is to keep documented to support, before running the `results|process`. The idea of the "checklist" document is to be a helper of the "expected" and signed off by the partner VS what was submitted.

I am also not sure if we need `OpenShift Version`, because it will be on the artifacts. The problem is the artifact will have the current version, in the case of `upgrade` mode, it will be the target, the tool will need to discover the previous release somewhere to review if the partner was certified on the 4.y-1.

How do you think we can proceed here?